docs/using_lagoon/environment_variables.md

```diff
@@ -67,6 +67,27 @@ mutation addBuildtimeEnv {
   }
 }
 ```
+### Docker Registry Environment Variables (Lagoon API)
+
+Docker Registry Environment Variables are only available during a Build and are used when attempting to log in to a private registry. It is used to store the password for the user defined in [Specials » `private-registries`](./lagoon_yml.md). They can be applied at the project or environment level.
+
+This defines a project wide docker registry variable (available in all environments) for the project with id `463`:
+```
+mutation addDockerRegistryEnv {
+  addEnvVariable(input:{type:PROJECT, typeId:463, scope:DOCKER, name:"MY_OWN_REGISTRY_PASSWORD", value:"MySecretPassword"}) {
```

Comment:
I'm not sure about the name `DOCKER` here, I would have called it `REGISTRY`? also I think we should slowly get rid of Docker all together, as it might not exist for much longer, so maybe even `CONTAINER_REGISTRY`?

Reply:
Yep, good idea. I think go with `CONTAINER_REGISTRY`. I'll make changes